orchestra/utils/task_lifecycle.py

```diff
@@ -1174,6 +1178,12 @@ def create_subsequent_tasks(project):
     completed_step_slugs = set(completed_tasks.values_list('step__slug',
                                                            flat=True))
 
+    if len(completed_step_slugs) == len(all_steps):
```

Comment:
i always get worried about throwing random `return`s into the middle of long functions---can you get rid of the return and put this at the end of the function?